download_cache.nf

```diff
@@ -200,7 +184,9 @@ process BuildCache_VEP {
   """
 }
 
-caddFileToDownload = (params.cadd_version) && (params.genome == "GRCh37" || params.genome == "GRCh38") ?
+vep_cache_out = vep_cache_out.dump(tag: 'vep_cache_out')
+
+caddFileToDownload = (params.cadd_cache && params.cadd_version) && (params.genome == "GRCh37" || params.genome == "GRCh38") ?
```

Comment:
Shouldn't this be lower-case, for consistency? I guess not important if back-end only and complex to change.

Reply:
Params are already all snake_cased, did not want to supercharge the PR with snake_case variables too, planning to snake_case everything with the next release that will include DSL 2